javascript/javascript1/week1/Houseypricey.js

```diff
@@ -0,0 +1,28 @@
+//peter house calculation
+var width = 8;
+var height = 10;
+var depth = 10;
+var gardenSizeInM2 = 100;
+var actualhousePrise = 2500000;
+var volumeInMeters;
+var housePrice;
+volumeInMeters = width * height * depth;
+housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+if (housePrice > actualhousePrise) {
+console.log ("peter is paying exess price");}
+else
+{console.log("peter is paying less than the actual price");}
+//Julia house calculation
```

Comment:
you can use arrays to make a simple code like this let width=[8,5];
const depth=[10,11];
const height=[10,8];
const gardenSizeInM2=[100,70];
const actualHousePrice=[2500000 , 1000000];
const VolumeInMeters=[width[0]*deep[0]*height[0] , width[1]*deep[1]*height[1]]
const housePrice = 
[VolumeInMeters[0] * 2.5 * 1000 + gardenSizeInM2[0] * 300,
     VolumeInMeters[1] * 2.5 * 1000 + gardenSizeInM2[1] * 300];
then you can compare using their index values like this :
if(actualHousePrice[0]<=housePrice[0])